test/integration/targets/connection_aws_ssm/runme.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -eux
+
+CMD_ARGS=$*
```

Comment:
Preserve the script args as an array.

```suggestion
CMD_ARGS=("$@")
```

Reply:
Hi @mattclay ,
Fixed this issue in the commit https://github.com/ansible/ansible/pull/49652/commits/1bd4a9f2a38485603129cda99ab7324026ef9b7d